idseq_pipeline/cli.py

```diff
@@ -48,19 +48,19 @@
 
 from docopt import docopt
 
-from . import __version__ as VERSION
+from version import __version__ as VERSION
 
 
 def main():
     """Main CLI entrypoint."""
-    import idseq_pipeline.commands
+    import commands as idseq_pipeline_commands
     options = docopt(__doc__, version=VERSION)
 
     # Match user-entered command
     for (k, v) in options.items(): 
-        if hasattr(idseq_pipeline.commands, k) and v:
-            module = getattr(idseq_pipeline.commands, k)
-            idseq_pipeline.commands = getmembers(module, isclass)
```

Comment:
This line actually appears to have been modifying the loaded module, which is probably not intentional and slightly dicey.   I hope it was not intentional.   With the change everything seems to work fine.

Reply:
This is actually something I copied from skele-cli. Probably good to check if installation and CLI usage still works

Reply:
Heh ok!